src/create-ssr-algolia-client.ts

```diff
@@ -39,7 +39,7 @@ export function createSSRSearchClient({
   makeStateKey,
 }) {
   const client = algoliasearch(appId, apiKey, {});
-  client.addAlgoliaAgent(`angular-instantsearch ${VERSION}`);
+  client.addAlgoliaAgent(`angular-instantsearch (${VERSION})`);
```

Comment:
We can do the same than for the React version have a dedicated user agent for the server.

Reply:
@samouss @francoischalifour  `angular-instantsearch-server` looks good?

Reply:
Yes, it's the same version.

Reply:
Do we want to track the Node version?

Reply:
Not sure it's really useful in that case since we are bound to the Angular support of Node.

Reply:
the node version itself could be indicated by the JS client as noted by @samouss in the client PR

Reply:
Done